File: Java-Programs/Number Based/Leap_Year.java
// Check a number is Leap or Not
import java.util.Scanner;

public class Leap_Year {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Please enter a value");
        int Num = sc.nextInt();
        sc.close();

        if (Num % 4 == 0) {
            if (Num % 100 == 0) {
                if (Num % 400 == 0) {
                    System.out.println("leap year");
                } else {
                    System.out.println("not a leap year");
                }
                
            } else {
                System.out.println("leaap year");
            }

        } else {
            System.out.println("not a leap year");
        }
    }
}

File: PHP-Programs/Leap Year/leapYear.php
<html>
<head>Check Leap Year</head>
<body>
<form method="post" action="<?php echo $_SERVER["PHP_SELF"]; ?>">
        <label for="input">Enter your input:</label>
        <input type="number" name="Num" id="input">
        <input type="submit" value="Submit">
</form>
<?php
if ($_SERVER["REQUEST_METHOD"] == "POST" && !empty($_REQUEST['Num'])) {
function LeapYear($Num){
    if($Num % 4 == 0){
        if($Num % 100 == 0 && $Num % 400 != 0){
            return false;
        }else{
            return true;
        }
    }
}
$Num=$_REQUEST['Num'];
if(LeapYear($Num)==true){
    echo "Leap Year";
}else{
    echo "Not a Leap Year";
}
}
?>
</body>
</html>